include/openmc/position.h

```diff
@@ -210,4 +211,14 @@ using Direction = Position;
 
 } // namespace openmc
 
+template <>
+struct fmt::formatter<openmc::Position> : formatter<std::string> {
+  template <typename FormatContext>
+  auto
+  format(const openmc::Position & pos, FormatContext & ctx)
+  {
+    return fmt::formatter<std::string>::format(fmt::format("({}, {}, {})", pos.x, pos.y, pos.z), ctx);
+  }
+};
```

Comment:
Some formatting changes from our `.clang-format` file:
```suggestion
template<>
struct fmt::formatter<openmc::Position> : formatter<std::string> {
  template<typename FormatContext>
  auto format(const openmc::Position& pos, FormatContext& ctx)
  {
    return fmt::formatter<std::string>::format(
      fmt::format("({}, {}, {})", pos.x, pos.y, pos.z), ctx);
  }
};
```

Reply:
clang-formatting applied. Should have noticed the clang-format file...